src/Kunstmaan/GeneratorBundle/Resources/config/services.yml

```diff
@@ -47,3 +47,7 @@ services:
       tags:
         - { name: console.command }
 
+    Kunstmaan\GeneratorBundle\Command\InstallCommand:
+        arguments: ['%kernel.root_dir%']
```

Comment:
Change this to `%kernel.project_dir%` as the root_dir parameter is deprecated in symfony 4.2